recipes/fmt/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "fmt" %}
+{% set version = "6.0.0" %}
+{% set sha256 = "f1907a58d5e86e6c382e51441d92ad9e23aea63827ba47fd647eacc0d3a16c78" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/fmtlib/fmt/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win]
```

Comment:
```suggestion
  skip: True  # [win and vc<14]
```

Reply:
@isuruf Why is this needed if `win` is entirely skipped?